.github/mergify.yml

```diff
@@ -48,17 +48,17 @@ pull_request_rules:
       # CI workflows approvals.
       - check-success=test-suite-success
       - check-success=local-testnet-success
-      - "#review-requested > 0"
+      # Update the label only if there are no more change requests from any reviewers and no unresolved threads.
+      # This rule ensures that a PR with passing CI can be marked as `waiting-on-author`.
+      - or:
```

Comment:
did you want this to be an `and`? the description implies it's an `and`, but then there's an `or` here